mwdb/web/src/components/Profile/Views/ProfileGroups.jsx

```diff
@@ -77,7 +75,7 @@ export default function ProfileGroups() {
             </div>
             <table className="table table-bordered wrap-table">
                 <tbody>
-                    {workspaces
+                    {workspaces && workspaces
```

Comment:
```suggestion
                    {workspaces
```
Artifact from tests / uneccesary